### PR TITLE
Minor fixes for oVirt modules

### PR DESCRIPTION
**Pull request by machacekondra**
##### SUMMARY

This PR includes patches which fixes few minor bugs for oVirt modules.

* cloud: ovirt: add post-copy scheduling policy
* cloud: ovirt: only image disk can be sparsified

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

ovirt_clusters
ovirt_disks

##### ANSIBLE VERSION

```
2.3.0
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by resmo** [comment]
shipit

**Comment by machacekondra** [comment]
shipit

**Comment by machacekondra** [comment]
shipit

**Comment by machacekondra** [comment]
@ryansb

**Comment by machacekondra** [comment]
Can you please merge. It's fixing a bug, which is causing failing of our integration tests for those modules:

 http://jenkins.ovirt.org/job/ovirt_master-ansible-system-tests/

File: lib/ansible/modules/cloud/ovirt/ovirt_clusters.py [hunk 1]
```diff
@@ -173,6 +173,8 @@
             - "C(legacy) - Legacy behavior of 3.6 version."
             - "C(minimal_downtime) - Virtual machines should not experience any significant downtime."
             - "C(suspend_workload) - Virtual machines may experience a more significant downtime."
+            - "C(post_copy) - Virtual machines should not experience any significant downtime.
+               If the VM migration is not converging for a long time, the migration will be switched to post-copy."
         choices: ['legacy', 'minimal_downtime', 'suspend_workload']
```
Comment (pilou-) [inline lib/ansible/modules/cloud/ovirt/ovirt_clusters.py]: `choices` : the new value must be added.

Comment (machacekondra) [reply lib/ansible/modules/cloud/ovirt/ovirt_clusters.py]: Fixed, thanks.

File: lib/ansible/modules/cloud/ovirt/ovirt_clusters.py [hunk 2]
```diff
@@ -173,7 +173,9 @@
             - "C(legacy) - Legacy behavior of 3.6 version."
             - "C(minimal_downtime) - Virtual machines should not experience any significant downtime."
             - "C(suspend_workload) - Virtual machines may experience a more significant downtime."
```
Comment (pilou-) [inline lib/ansible/modules/cloud/ovirt/ovirt_clusters.py:175]: You should mention that `post_copy` was added in 2.4.

Comment (machacekondra) [reply lib/ansible/modules/cloud/ovirt/ovirt_clusters.py:175]: Makes sense, thanks.

**Review by mwperina — APPROVED** [review]
👍

**Comment by machacekondra** [comment]
shipit

**Comment by machacekondra** [comment]
Can you please merge our infra is still failing:

 - https://ovirt-jira.atlassian.net/browse/OVIRT-1356
 - http://jenkins.ovirt.org/job/ovirt_master-ansible-system-tests/

**Review by ryansb — APPROVED** [review]

**Pull request merged**